src/backend/XmlHttpRequest.ts

```diff
@@ -97,6 +104,7 @@ export default class XmlHttpRequest extends BaseFileSystem implements FileSystem
 
   public readonly prefixUrl: string;
   private _index: FileIndex<{}>;
+  private _useFetch: boolean;
```

Comment:
> As in previous comments,

These must not be showing up in the order you wrote them. :/ Oh well, I'll figure it out eventually! It's like conversations on Twitter...